### PR TITLE
Feat: Initial pass at groupers

### LINKED ISSUE #37: Add groupers
- [ ] Gene
- [ ] Drug
- [ ] Interaction

### PR DESCRIPTION
**Pull request by jsstevenson**
Close #37 
* Basic functionality for drug, gene, and interaction groupers
* Make some small DB structure alterations to accommodate better data organization
* Also some unrelated file cleanup (git crime)

### CODE REVIEW

File: server/db/migrate/20220408032834_change_gene_id_field.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ChangeGeneIdField < ActiveRecord::Migration[6.1]
+  def self.up
+    rename_column :genes, :entrez_id, :concept_id
+  end
+
+  def self.down
+    rename_column :genes, :concept_id, :entrez_id
+  end
+end
```
Comment (mcannon068nw) [inline server/db/migrate/20220408032834_change_gene_id_field.rb:9]: does up and down correspond to the jump from "claim" to normalized record status?

File: server/lib/genome/groupers/interaction_grouper.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      def self.run(group_from_scratch=false)
+        if group_from_scratch
+          ActiveRecord::Base.transaction do
+            puts 'reset members'
+            reset_members
+            puts 'add members'
+            add_members(group_from_scratch)
+          end
+        else
+          puts 'add members'
```
Comment (mcannon068nw) [inline server/lib/genome/groupers/interaction_grouper.rb:13]: are these puts statements for debugging or for informing during run

File: server/lib/utils/logging.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  module Logging
+    def self.without_sql
+      old_logger = ActiveRecord::Base.logger
+      ActiveRecord::Base.logger = nil
+      yield
+      ensure
+        ActiveRecord::Base.logger = old_logger
+    end
+  end
+end
```
Comment (mcannon068nw) [inline server/lib/utils/logging.rb:12]: Yes! Logs!

**Review by mcannon068nw — APPROVED** [review]

File: server/db/migrate/20220408032834_change_gene_id_field.rb (hunk 1 above)
Comment (jsstevenson) [reply server/db/migrate/20220408032834_change_gene_id_field.rb:9]: This is actually Ruby migration lingo, the idea being that when you write a structural change to the DB (up) you can also write a routine to undo it (down)

**Pull request merged**